WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/TemplatePreviewViewController.swift

```diff
@@ -100,10 +100,13 @@ class TemplatePreviewViewController: UIViewController, NoResultsViewHost, UIPopo
     }
 
     private func styleButtons() {
+        let isLastSiteCreationStep = ABTest.siteNameV1.variation == .treatment(nil) && FeatureFlag.siteName.enabled
+        let mainButtonTitle = isLastSiteCreationStep ? NSLocalizedString("Create site", comment: "Title for the button to progress with creating the site with the selected design")
+            : NSLocalizedString("Choose", comment: "Title for the button to progress with the selected site homepage design")
```

Comment:
nit: the comment should have a period at the end.

Reply:
Handled with babc542445c63b38afa12643abe19746e3dc30cb 👍